### PR TITLE
github: paralellize action execution;

### PR DESCRIPTION
**Pull request by ss89**

### CODE REVIEW

**Comment by ss89** [comment]
@selm0 mind a quick review?

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -9,40 +9,97 @@ on:
...
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: go get -v -t -d ./...
+
       - name: Integration tests
-        run: go test -p 1 --tags='integration,fixtures' ./test/...
+        run: go test -p 3 --tags='integration,fixtures' ./test/...
```
Comment (selm0) [inline .github/workflows/go.yml:105]: Why 3? Maybe we can rely on GOMAXPROCS?

**Review by selm0 — CHANGES_REQUESTED** [review]
Please double check the number of go procs for the integration test.

File: .github/workflows/go.yml (hunk 1 above)
Comment (applike-ss) [reply .github/workflows/go.yml:105]: i tried, but then in fact the github runner is so busy that it can't finish the tests at all

**Review by selm0 — APPROVED** [review]

**Pull request merged**